src/tests/messageUtils.ts

```diff
@@ -0,0 +1,264 @@
+/**
+ * Copyright (c) 2018-2023, BOTLabs GmbH.
+ *
+ * This source code is licensed under the BSD 4-Clause "Original" license
+ * found in the LICENSE file in the root directory of this source tree.
+ */
+
+import {
+  KiltKeyringPair,
+  KeyringPair,
+  KiltEncryptionKeypair,
+  DidDocument,
+  NewLightDidVerificationKey,
+  KeyRelationship,
+  DidServiceEndpoint,
+  DidKey,
+  DidVerificationKey,
+  DecryptCallback,
+  EncryptCallback,
+} from '@kiltprotocol/types'
+
+import { Did, SignCallback } from '@kiltprotocol/sdk-js'
+import { Crypto } from '@kiltprotocol/utils'
+import { blake2AsU8a, blake2AsHex } from '@polkadot/util-crypto'
+
+/**
+ * Generates a callback that can be used for signing.
+ *
+ * @param keypair The keypair to use for signing.
+ * @returns The callback.
+ */
+export function makeSignCallback(keypair: KeyringPair): KeyToolSignCallback {
```

Comment:
Don't we have make call backs inside other utils not inside the messaging? Do we want to make these global exports that people can use them etc.?